src/Packetery/Core/Api/Soap/Response/CreatePacket.php

```diff
@@ -94,12 +94,31 @@ public function getValidationErrors(): ?array {
 	/**
 	 * Gets all errors as string.
 	 *
+	 * @param bool $prependFaultString Prepend fault string.
+	 *
 	 * @return string
 	 */
-	public function getErrorsAsString(): string {
+	public function getErrorsAsString( bool $prependFaultString = true ): string {
 		$allErrors = $this->validationErrors;
-		array_unshift( $allErrors, $this->getFaultString() );
+		if ( $prependFaultString ) {
+			array_unshift( $allErrors, $this->getFaultString() );
+		}
 
 		return implode( ', ', $allErrors );
 	}
+
+	/**
+	 * Returns error message, which is made of validation errors or fault string, otherwise null
+	 *
+	 * @return string|null
+	 */
+	public function getErrorMessage(): ?string {
+		if ( ! $this->hasFault() ) {
```

Comment:
Není tato podmínka zbytečná s ohledem na to, kde se metoda volá ?

Reply:
zrefaktoroval jsem submitPacket , tak ze jiz neni zbytecna.